src/__tests__/env.spec.ts

```diff
@@ -0,0 +1,34 @@
+const execa = require("execa")
```

Comment:
If a dependency is not in use, remove it from package.json as well @anuragpandey1115